pkg/datapath/loader/base.go

```diff
@@ -161,6 +167,24 @@ func cleanIngressQdisc(devices []string) error {
 	return nil
 }
 
+func linkList() ([]netlink.Link, error) {
+	for try := 0; try < linkListMaxTries; try++ {
+		links, err := netlink.LinkList()
+		if err == nil {
+			return links, nil
+		}
+		sleep := backoff.CalculateDuration(
+			linkListMinRetryInterval,
+			linkListMaxRetryInterval,
+			2.0,
+			false,
+			try)
+		time.Sleep(sleep)
+	}
+
+	return nil, errors.New("Could not load links")
```

Comment:
Can you use fmt.Errorf and then pass the original error as well with %w so that the original error can be debugged if it comes to this point.